setup.py

```diff
@@ -4,7 +4,7 @@
 
 __version__ = "0.11"
 
-tests_require = ['pytest', 'mypy', 'pycodestyle']
+tests_require = ['pytest','pytest-asyncio', 'mypy', 'pycodestyle']
```

Comment:
what does the pytest async module do that we cant do ourselves?

Reply:
pytest-asyncio is a good plugin for pytest to handle Coroutines. I haven't found such a thing for Unittest (and Unittest is supporting asyncio stuff only >=Python3.8).